CMakeLists.txt

```diff
@@ -57,18 +57,19 @@ option(ENABLE_ADJUSTABLEHUD_MOD "Install the build‑in “AdjustableHud mod”
 
 if(ENABLE_TFE)
 	add_executable(tfe)
-	set_target_properties(tfe PROPERTIES OUTPUT_NAME "theforceengine")
+	set_target_properties(tfe PROPERTIES OUTPUT_NAME "theforceengine_target") # rename the binary to "theforceengine_target" to avoid conflicts with the directory name on mac os x
```

Comment:
I don't know how @luciusDXL feels about this, maybe use "theforceengine_osx" and "theforceengine_linux" but _target sounds just wrong to me.

Reply:
Yes, I agree. It was just a quick stopgap solution to avoid the name conflict. What do you think about autoappending CMAKE's the platform name:

```
set(FINAL_OUTPUT_NAME "theforceengine_${CMAKE_SYSTEM_NAME}")
set_target_properties(tfe PROPERTIES OUTPUT_NAME "${FINAL_OUTPUT_NAME}")
```